gistic/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:14.04
```

Comment:
Will this work with ubuntu 16 or 18? 14 doesn't have LTS anymore I don't think...

Reply:
Not working on 16 and 18

```
E: Unable to locate package libXext6
E: Unable to locate package libXt6
E: Unable to locate package libXmu6
E: Unable to locate package libXp6
```

Reply:
Yup, I see,t hanks!